store/sqlstore/user_store.go

```diff
@@ -7,16 +7,16 @@ import (
 	"database/sql"
 	"encoding/json"
 	"fmt"
-	"net/http"
 	"sort"
 	"strings"
 
-	sq "github.com/Masterminds/squirrel"
 	"github.com/mattermost/gorp"
```

Comment:
Please join this with the 3rd group.

Reply:
I can't see this change, it is already added in the 3rd group